src/main/java/org/kairosdb/core/http/rest/MetricsResource.java

```diff
@@ -104,6 +127,9 @@
 	@Inject
 	private Tracer tracer;
 
+	private TimeLimiter limiter;
+	private ThreadPoolExecutor executor;
```

Comment:
Do we use this reference to the executor anywhere else in the class except to initialize the limiter in the constructor?  If not we should delete this.

Reply:
this executor is used to get the metrics about the thread pool in the other methods